invenio_communities/config.py

```diff
@@ -21,8 +21,8 @@
 
 COMMUNITIES_ROUTES = {
     "frontpage": "/communities",
-    "search": "/communities/search",
-    "new": "/communities/new",
+    "search": "/search/communities/all",
```

Comment:
What is the impact of these changes?

Reply:
added my thoughts here https://github.com/inveniosoftware/invenio-communities/pull/1210#discussion_r1784500810

Reply:
@sakshamarora1 did you check if we don't hardcode these URLs somewhere in the UI?

Reply:
There are, I will update. But I am still not sure if we go ahead and with which change. Waiting for more people to give their opinions.

Reply:
Given that the impact and mitigations (besides internally changing links in the codebase) are not easy to evaluate (e.g. existing public docs for RDM instances, having to communicate this in release notes, providing a transition, etc.) and can't be fixed similar to the `/communities/<slug> -> /communities/<slug>/records` URL redirect we did for the community records, I'm leaning toward the reserved keywords option.

That can be either done by manually maintaining the list of keywords or automatically calculating it from the registered routes.